framework/core/src/Http/Filter/UserFilter.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Http\Filter;
+
+use Flarum\Filter\FilterInterface;
+use Flarum\Filter\FilterState;
+
+class UserFilter implements FilterInterface
```

Comment:
Feels a bit odd that nothing in this file mentions / relates to access tokens

Reply:
If it wasn't a breaking change, it would make sense to move access token domain to its own namespace.